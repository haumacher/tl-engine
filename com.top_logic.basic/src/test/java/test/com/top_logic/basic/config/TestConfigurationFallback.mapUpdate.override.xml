<?xml version='1.0' encoding='utf-8'?>
<fallback-test xmlns:config="http://www.top-logic.com/ns/config/6.0">
	<b-config intValue="15">
		<a-map-configs config:override="true">
			<aConfig
				config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
				intValue="15" stringValue="four" />
		</a-map-configs>
	</b-config>
</fallback-test>