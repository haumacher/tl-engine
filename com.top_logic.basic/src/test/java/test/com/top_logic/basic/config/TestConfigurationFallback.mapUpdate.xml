<?xml version='1.0' encoding='utf-8'?>
<fallback-test xmlns:config="http://www.top-logic.com/ns/config/6.0">
	<b-config stringValue="oldStringValue">
		<a-map-configs>
			<aConfig
				config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$B"
				intValue="15" stringValue="one"></aConfig>
			<aConfig
				config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$B"
				stringValue="two" intValue="-2"></aConfig>
			<aConfig
				config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$B"
				stringValue="three" intValue="-1"></aConfig>
		</a-map-configs>
	</b-config>
</fallback-test>