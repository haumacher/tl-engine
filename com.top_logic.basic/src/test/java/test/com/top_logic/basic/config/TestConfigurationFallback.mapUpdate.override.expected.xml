<?xml version='1.0' encoding='utf-8'?>
<fallback-test xmlns:config="http://www.top-logic.com/ns/config/6.0">
	<b-config stringValue="oldStringValue" intValue="15">
		<a-map-configs>
			<aConfig
				config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
				intValue="15" stringValue="four" />
		</a-map-configs>
	</b-config>
</fallback-test>